src/hooks/useVersionTaskStatusSelect.ts

```diff
@@ -206,29 +206,40 @@ const reduceBuildVariants = (filterDetails: reduceInput) => {
 
   const statuses = new Set(versionStatusFilterTerm);
   const baseStatuses = new Set(baseStatusFilterTerm);
+  // if 1 of the 2 filters is empty, ignore the empty filter
+  const hasFilter =
+    !!versionStatusFilterTerm?.length || !!baseStatusFilterTerm?.length;
+
+  function hasStatus(status) {
+    return versionStatusFilterTerm?.length ? statuses.has(status) : true;
```

Comment:
it seems like `versionStatusesFilterTerm` is always defined so the `?` is not necessary.

Reply:
It's needed because when it first loads it's undefined.